EventHighway.Core/Clients/ListenerEvents/V2/ListenerEventV2sClient.cs

```diff
@@ -42,6 +43,43 @@ public async ValueTask<IQueryable<ListenerEventV2>> RetrieveAllListenerEventV2sA
             }
         }
 
+        public async ValueTask<ListenerEventV2> RemoveListenerEventV2ByIdAsync(Guid listenerEventV2Id)
+        {
+            try
+            {
+                return await this.eventListenerV2OrchestrationService.RemoveListenerEventV2ByIdAsync(
+                    listenerEventV2Id);
+            }
+            catch (EventListenerV2OrchestrationDependencyValidationException
+                eventListenerV2OrchestrationDependencyValidationException)
+            {
+                throw CreateListenerEventV2ClientDependencyValidationException(
+                    eventListenerV2OrchestrationDependencyValidationException.InnerException
+                        as Xeption);
+            }
+            catch (EventListenerV2OrchestrationValidationException
+                eventListenerV2OrchestrationValidationException)
+            {
+                throw CreateListenerEventV2ClientDependencyValidationException(
+                    eventListenerV2OrchestrationValidationException.InnerException
+                        as Xeption);
+            }
```

Comment:
```suggestion
catch (EventListenerV2OrchestrationValidationException
                eventListenerV2OrchestrationValidationException)
            {
                throw CreateListenerEventV2ClientDependencyValidationException(
                    eventListenerV2OrchestrationValidationException.InnerException
                        as Xeption);
            }
            catch (EventListenerV2OrchestrationDependencyValidationException
                eventListenerV2OrchestrationDependencyValidationException)
            {
                throw CreateListenerEventV2ClientDependencyValidationException(
                    eventListenerV2OrchestrationDependencyValidationException.InnerException
                        as Xeption);
            }
```

ValidationException should be handled before dependencyValidation